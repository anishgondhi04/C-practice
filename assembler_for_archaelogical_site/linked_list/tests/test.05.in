2
list 0 new
list 1 new
list 0 build 5 alpha beta gamma delta epsilon
list 0 overlap 1
list 0 splice 1
list 1 print
list 1 new
list 1 overlap 0
list 1 splice 0
list 0 print
list 0 new
list 0 overlap 0
list 0 splice 0
list 1 overlap 1
list 1 splice 1
end
