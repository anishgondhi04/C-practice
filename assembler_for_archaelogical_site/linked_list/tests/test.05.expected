List 0: new list
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 0: build
 [ alpha beta gamma delta epsilon ] (5 items)
List 0: Overlap between lists 0 and 1 is 0
List 0: List 1 spliced to the end of list 0
 [ alpha beta gamma delta epsilon ] (5 items)
List 1: print
 [ ] (0 items)
List 1: new list
 [ ] (0 items)
List 1: Overlap between lists 1 and 0 is 0
List 1: List 0 spliced to the end of list 1
 [ alpha beta gamma delta epsilon ] (5 items)
List 0: print
 [ ] (0 items)
List 0: new list
 [ ] (0 items)
List 0: Overlap between lists 0 and 0 is 0
List 0: List 0 spliced to the end of list 0
 [ ] (0 items)
List 1: Overlap between lists 1 and 1 is 5
List 1: List 1 spliced to the end of list 1
 [ alpha beta gamma delta epsilon ] (5 items)
